### DISCUSSION TITLE
Can i create persistent layout?

### DISCUSSION THREAD

**Discussion by fahmiidris** [markdoc/markdoc#61] [Q&A] [answered]
[Reference](https://adamwathan.me/2019/10/17/persistent-layout-patterns-in-nextjs/)

I'm working on a project with next js and tailwindcss which happens to have a lot of different layouts.

I followed the technique exemplified in the reference, and it worked in a regular JSX file.

Then how to apply it to the MD file?

**Comment by mfix-stripe** [comment, marked as answer]
@fahmiidris-labs you can create a layout in your `_app.js` file. 

Here is the docs for that: https://markdoc.io/docs/nextjs#layouts, and here is an example of how we did it for our own documentation site: https://github.com/markdoc/docs/blob/main/pages/_app.js
